package.json

```diff
@@ -29,5 +29,7 @@
     "webpack": "^4.41.5",
     "webpack-cli": "^3.3.10"
   },
-  "dependencies": {}
```

Comment:
I am unable to run `npm run build` because you're missing the script.
![image](https://user-images.githubusercontent.com/31897434/74288524-decb8100-4d2c-11ea-8d9c-a160d948074b.png)
You should have something like; 
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

Reply:
I am sure I have just seen production - I am still not sure the names carry a lot of meaning, but I have updated it to build.

Reply:
Okay, check again. I added webpack-dev-server.  Line 32 is not correct though - the dev deps which include the build tools cannot be there, thanks for pointing out though.